mautic/mautic
# [1198] - filter by "Email received" where you can select a specific e…

Description: 
In the Leads / Manage Lists / when adding / editing a list, you can specify a new filter that is called "Email received". Select one email and save the filter. 
You will then be able to generate a list that will contain leads that did receive that specific email. 

Steps to test: 
- before be sure that you have some Email already created and some of the leads already received the emails. 
- go to Leads / Manage Lists
- create a new Lead Smart List (give it a name, alias and a description)
- move to filters tab and in there select "Email received"
- select Includin / excluding
- select one / more emails (email subjects are displayed) from the list

click apply and then save and close. 

run the command to update the leads for this new list that you just created.
 php /path/to/mautic/app/console mautic:leadlists:update --env=prod

You should have inside the leads that received the email you just selected.
